src/searchengine/nova3/helpers.py

```diff
@@ -34,13 +34,17 @@
 import re
 import socket
 import socks
+import ssl
 import tempfile
 import urllib.error
 import urllib.parse
 import urllib.request
 
+# Some sites have invalid certificates
+ssl._create_default_https_context = ssl._create_unverified_context
+
 # Some sites blocks default python User-agent
-user_agent = 'Mozilla/5.0 (X11; Linux i686; rv:38.0) Gecko/20100101 Firefox/38.0'
+user_agent = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:64.0) Gecko/20100101 Firefox/64.0'
```

Comment:
maybe use the firefox esr? `60.0` just some thoughts.

Reply:
Any version works well, I'm just using the last.